Documentation/contributing/testing/unit.rst

```diff
@@ -28,13 +28,19 @@ both etcd and consul. To start the local instances, run:
 Running all tests
 ^^^^^^^^^^^^^^^^^
 
-To run unit tests over the entire repository, run the following command in the
-project root directory:
+To run integration tests over the entire repository, run the following command
```

Comment:
I know that this PR is a step forward in terms of naming the test suites correctly, but having `integration` here followed by `make unit-tests` would scream typo for me when reading the docs.

Would renaming the make target be possible, or is this out of scope for this PR?

Maybe we can create a `integration-tests` target as an alias for `unit-tests` so that the docs look good and we maintain backwards compatibility?

Reply:
Done. I've renamed the `unit-tests` target to `integration-tests`.